roles/uperf-bench/templates/server.yml.j2

```diff
@@ -18,6 +18,9 @@ spec:
   containers:
   - name: benchmark
     image: "quay.io/cloud-bulldozer/uperf:latest"
+{% if uperf.server_resources is defined %}
+    resources: {{ uperf.server_resources | to_json }}
```

Comment:
I'm not sure if the jinja2 templating will be happy with this, kicked off CI run lets see